### ISSUE TITLE
Irritating (takes huge time for setup)

### ISSUE THREAD

**Issue by sakibnjr** [modded-ubuntu/modded-ubuntu#143]

**Comment by BDhackers009** [comment]
Yes it takes time to install all the required packages. The script installs everything interactively to give the user vast choices and it also reduces or increases the installation size accordingly. If you want to install ubuntu with a large 1+ GB of rootfs then search for udroid.

**Comment by SUMITDEY69** [comment]
Enjoy the process, it's worth waiting 😁

**Comment by BDhackers009** [comment]
> Enjoy the process, it's worth waiting 😁

Happy staying 🖤
